configure.ac

```diff
@@ -188,6 +188,17 @@ AC_SUBST(BLUEZ_CFLAGS)
 AC_SUBST(BLUEZ_LIBS)
 AM_CONDITIONAL(BLUETOOTH, test "${enable_bluetooth}" != "no")
 
+AC_ARG_ENABLE(cares, AC_HELP_STRING([--disable-cares],
+				[disable DNS client using c-ares library]),
+						[enable_cares=${enableval}])
+if (test "${enable_cares}" != "no"); then
+	PKG_CHECK_MODULES(CARES, libcares, dummy=yes,
```

Comment:
As all other settings in the configure file, support for c-ares is on by default, and you can disable it using --disable-cares. So you do not need --enable-cares. enable_cares is a shell name that autoconfigure sets when running "configure" script, and as such needs the underscore (and you can see underscores for these variables in all of configure.ac).

I suggest removing that --enable_cares and trying again, although I will give it a try too.